test/framework/filetools.py

```diff
@@ -189,11 +188,14 @@ def test_download_file(self):
 
         # install broken proxy handler for opening local files
         # this should make urllib2.urlopen use this broken proxy for downloading from a file:// URL
-        proxy_handler = urllib2.ProxyHandler({'file': 'file://%s/nosuchfile' % test_dir})
+        proxy_handler = urllib2.ProxyHandler({'https': 'http://%s/nosuchfile' % test_dir})
         urllib2.install_opener(urllib2.build_opener(proxy_handler))
 
         # downloading over a broken proxy results in None return value (failed download)
         # this tests whether proxies are taken into account by download_file
+        fn = "robots.txt"
+        source_url = "https://jenkins1.ugent.be/"
+        target_location = os.path.join(self.test_buildpath, 'some', 'subdir', fn)
         self.assertEqual(ft.download_file(fn, source_url, target_location), None, "download over broken proxy fails")
```

Comment:
the new system uses the protocol to figure out what it is doing, for file:// it uses shutil.copy, so no urllib2, so no proxy, so this test is not testing what it should test.

If you want I can change this to use urllib2 for 'downloading' files, but it doesn't make as much sense...